### DISCUSSION TITLE
format of timestamp?

### DISCUSSION THREAD

**Discussion by surbhika-rastogi** [priyankark/PhonePi_SampleServer#17] [General]
Hi. Where can i get to know the format of timestamp used in the data logging?

**Comment by priyankark** [comment]
@surbhika-rastogi timestamp is the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC.
More here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now
